aip/general/0151.md

```diff
@@ -129,6 +129,20 @@ metadata message. The errors themselves **must** still be represented with a
 Changing either the `response_type` or `metadata_type` of a long-running operation
 is a breaking change.
 
+## Rationale
+
+The guidance for validate-only responses comes from a tension
+between clients, which benefit from "fully formed" operations that can
+be treated uniformly, and servers, which don't wish to maintain
```

Comment:
Exactly! I think this is basically summed up by the idea that validate-only is an edge case to the "contract" provided by LROs - you execute an RPC, it produces an Operation, you can "wait" for that Operation to complete via the interface provided **always**. Carving out a case where one *does not* do the "normal thing" breaks the contract, creates a new case for *all* clients to handle specifically, and further complicates the contract. The simple/best thing, for the benefit of *users* and at the cost of *service owners*, is to fulfill the contract.

Admittedly, the state part of it is understandable.

`</rant>`

LGTM this is a good compromise. I think clients shouldn't need the `name` if `done = true`, clients should be resilient to that.